test/run_test.py

```diff
@@ -573,11 +573,7 @@ def run_test(
 
 def try_set_cpp_stack_traces(env, command, set=True):
     # Print full c++ stack traces during retries
-    # Don't do it for macos inductor tests as it makes them
-    # segfault for some reason
-    if not (
-        IS_MACOS and len(command) >= 2 and command[2].startswith(INDUCTOR_TEST_PREFIX)
-    ):
+    if not (len(command) >= 2 and command[2].startswith(INDUCTOR_TEST_PREFIX)):
```

Comment:
The above would disable torch inductor print, on both MACOS and ARM. 

The original intention was to disable MACOS's torch inductor tests' TORCH_SHOW_CPP_STACKTRACES. 

We would to enable them, so we could just delete the entire if command. 

Stacktrace would be printed across the board, and should have no issues. 

BTW, thank you for addressing in this PR. I meant to say in a follow-up PR.

Reply:
The import of IS_MACOS can be deleted too as this is the only occurence.

Reply:
>The import of IS_MACOS can be deleted too as this is the only occurrence.

Nope, it is not the only occurrence:
https://github.com/pytorch/pytorch/blob/fc61aae70f249c9b601fac6837df0e3ccef4f71f/test/run_test.py#L1268-L1289

Reply:
You are right!